### DISCUSSION TITLE
Can i change limit view and group by page ? it is possible?

### DISCUSSION THREAD

**Discussion by JoaoLima93** [nocodb/nocodb#6933] [Q&A] [answered]
How can i change limit view and group by page ? it is possible?

**Comment by dstala** [comment]
@JoaoLima93 Can you elaborate on your requirement? do you wish to change rows displayed per page in view & group-by?

**Reply by JoaoLima93** [reply]
Today nocodb shows by default 25 lines per page and 10 lines in grouped pages. I would like to increase this quantity to 100 lines each. We can't do it on screen, is there a config or internal varible in the application code that we can change to make this happen?

**Reply by dstala** [reply]
For Grid views, you will have to set `DB_QUERY_LIMIT_DEFAULT`; this defaults to 25;
https://docs.nocodb.com/getting-started/self-hosted/environment-variables

I am not sure if we have environment variable exposed for configuring grouped pages; I will let @mertmit address that.

**Reply by mertmit** [reply, marked as answer]
We don't have one yet but I will look into this, you can follow https://github.com/nocodb/nocodb/issues/6944
